From a850748d00958225f44d2a2637c2847523c9f590 Mon Sep 17 00:00:00 2001
From: Emmanuel Berthier <emmanuel.berthier@intel.com>
Date: Fri, 26 Jun 2015 19:37:20 +0200
Subject: [PATCH] EM/PMIC: AddressSanitizer: out of bounds access to
 wcove_bcu_id_table

This patch is to fix the buffer overflow in wcove_bcu_id_table
which is reported by the addressSanitizer.

Change-Id: I77448b97cd723d8b332d563c2c273b5a3da048f9
Tracked-On: https://jira01.devtools.intel.com/browse/IMINAN-40469
Signed-off-by: Emmanuel Berthier <emmanuel.berthier@intel.com>
Signed-off-by: Biju Chakkachattil <bijux.chakkachattil@intel.com>
---
 drivers/hwmon/intel_wcove_bcu.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/hwmon/intel_wcove_bcu.c b/drivers/hwmon/intel_wcove_bcu.c
index 2d13c86c4b65..9598488981d2 100644
--- a/drivers/hwmon/intel_wcove_bcu.c
+++ b/drivers/hwmon/intel_wcove_bcu.c
@@ -587,6 +587,7 @@ static int wcove_bcu_remove(struct platform_device *pdev)
 
 static const struct platform_device_id wcove_bcu_id_table[] = {
 	{DRIVER_NAME, 1},
+	{ }
 };
 
 static struct platform_driver wcpmic_bcu_driver = {
-- 
1.9.1

